src/modules/core/components/Numeral/Numeral.tsx

```diff
@@ -59,7 +59,7 @@ const Numeral = ({
     prefix,
     suffix,
     integerSeparator,
-    truncate,
+    truncate: convertedNum > 10 ** 4 ? 0 : truncate,
```

Comment:
I'd still want us to be able to control `truncate` manually.

So the "large number" logic should go in some other prop we just toggle true/false. (which should be true by default)

You can name it something like: `reducedOutput`